lib/mpl_toolkits/tests/test_mplot3d.py

```diff
@@ -491,3 +491,34 @@ def test_autoscale():
     ax.set_autoscalez_on(True)
     ax.plot([0, 2], [0, 2], [0, 2])
     assert ax.get_w_lims() == (0, 1, -.1, 1.1, -.4, 2.4)
+
+
+def invalid_axes_limits():
+    with pytest.raises(ValueError) as err:
+        plt.set_xlim3d(left=np.nan)
+    with pytest.raises(ValueError) as err:
+        plt.set_xlim3d(left=np.inf)
+    with pytest.raises(ValueError) as err:
+        plt.set_xlim3d(right=np.nan)
+    with pytest.raises(ValueError) as err:
+        plt.set_xlim3d(right=np.inf)
```

Comment:
These won't work because the default axes object do not have these methods. You will need to explicitly create the 3d axes object and call the methods on it.